test/compute_security_policies

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Before diving into the review. Could you please provide an example output here for testing?

Reply:
Sure, I have added an example output. I have also added some code in compute_security_policies.sh file to create a security policy, so that same output can be replicated by other developers as well